docs/plugin-proposal-dynamic-import.md

```diff
@@ -0,0 +1,103 @@
+---
+id: babel-plugin-proposal-dynamic-import
+title: "@babel/plugin-proposal-dynamic-import"
+sidebar_label: dynamic-import
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+> **NOTE**: This plugin is included in `@babel/preset-env`, in [ES2020](https://github.com/tc39/proposals/blob/master/finished-proposals.md).
```

Comment:
NIT: This can be admonition?

Reply:
It could be. Here I align the styles to other plugins: https://github.com/babel/website/blob/ea7ed1ee28638f4b32164cdebedde02f05b0bd69/docs/plugin-proposal-private-methods.md?plain=1#L7